targets/UnrealMarketplacePlugin/makeBp.js

```diff
@@ -296,3 +298,29 @@ function getAuthBools(tabbing, apiCall) {
 
     return output;
 }
+
+function getCustomApiAssignmentLogic(tabbing, apiCall) {
+    if (apiCall.name === "ExecuteFunction")
+    {
+        return tabbing + "// Check for local debugging\n"
+            + tabbing + "FString localApiServer = PlayFabCommon::PlayFabCommonUtils::GetLocalSettingsFileProperty(TEXT(\"LocalApiServer\"));\n"
+            + tabbing + "if (!localApiServer.IsEmpty())\n"
+            + tabbing + "{\n"
+            + tabbing + "    FString endpoint = TEXT(\"" + apiCall.url + "\");\n"
+            + tabbing + "    endpoint.RemoveFromStart(TEXT(\"/\"));\n"
+            + tabbing + "    FString url = localApiServer + endpoint;\n"
+            + tabbing + "    manager->PlayFabRequestFullURL = url;\n"
+            + tabbing + "}\n"
+    }
+}
+
+function getCustomApiActivationUrlLogic(api) {
+    if (api.name === "CloudScript")
```

Comment:
I don't think this is a strict enough check. api.name and apiCall.name would be specific enough.

Reply:
I believe that everywhere else you suggested this it works but not in this case.
The context is the API here not the call. There is no `Activate` method per endpoint, but rather the `CloudScript` API as a whole. As a result, what you are requesting cannot be done in this scenario specifically since there is no `apiCall` but rather just an `api`.

Reply:
Got it, thanks. I got overzealous with the copy paste. This specific one is resolved, but I think I'm right about the others.